docs/en/observability/synthetic-monitoring-visualizations.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[synthetic-monitoring-visualizations]]
+= Create synthetic monitoring visualizations
+
+experimental[]
+
+Based on the Uptime data you are sending to your deployment, you can create various visualizations
+relating to your monitor durations or pings over time data.
+
+[role="screenshot"]
+image::images/synthetic-monitoring-visualization.png[Synthetic monitoring visualization]
+
+. Go to *Observability > Uptime*.
+. On the *Overview* page, click *Analyze data*.
+. For data type, select *Synthetic Monitoring*.
+. Under **Report**, choose the type of data you want to analyze:
++
+|=== 
+
+| *Monitor duration* | The Uptime monitor duration time-series chart displays the timing for each check that {heartbeat} performed.
+
+The visualization helps you to gain insights into how quickly requests resolve by the targeted endpoint
+and give you a sense of how frequently a host or endpoint was down in your selected timespan.
+
+Hover over the chart to display
+crosshairs with specific metric data.
+
+| *Pings histogram* | The Uptime pings chart is a graphical representation of the check statuses over time.
+
+Hover over the chart to display crosshairs with specific metric data.
+
+|===
+
+. Let's define your report. Set the time filter as an absolute or relative time.
+. Select the monitor name and/or the URL. You can make multiple selections.
+. For a *monitor duration* report, you can specify the percentile to calculate and filter the chart by `average`, `median`,
+`75th`, `90th`, `95th`, or `99th`.
++
+NOTE: A percentile represents the value at which a certain percentage of observed values occur. For example, the 95th percentile is the
+value that is greater than 95% of the observed values.
+
+. From the *Chart type* dropdown, choose the aggregation-based visualization you want to view:
+
+* Bar vertical
+* Bar horizontal
+* Bar vertical stacked
+* Area
+* Area stacked
+* Line
+
+. Filter the type of data you want to examine.
++
+You can filter by `monitor type`, `observer location`, and any `tags` you previously created for a *monitor duration* report.
++
+You can filter by `observer location`, `monitor type`, and any `tags` you previously created for a *pings histogram* report.
+
+. To further segment the data and view multiple data series on the chart, you can specify a *breakdown*.
++
+For a *monitor duration* report, you can segment data by `location`, `ID`, or `port`.
++
+For a *pings histogram* report, you can segment data by `observer location` or `monitor type`.
+
+To customise the visualisation further, click *Open in Lens* where you can
```

Comment:
I'd change these to American English, especially since you use the AE "visualization" in the next sentence

```suggestion
To customize the visualization further, click *Open in Lens* where you can
```

Reply:
Ah, I'm a creature of habit :-)

Reply:
:)